pennylane/templates/embeddings.py

```diff
@@ -0,0 +1,266 @@
+# Copyright 2018 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Embeddings
+==========
+
+**Module name:** :mod:`pennylane.templates.embeddings`
+
+.. currentmodule:: pennylane.templates.embeddings
+
+This module provides quantum circuit architectures that can embed features into a quantum state.
+The features are associated with gate parameters (see also Schuld & Killoran 2019
+:cite:`schuld2018quantum`).
+
+Qubit architectures
+-------------------
+
+Angle embedding
```

Comment:
It was not directly clear to me that Pauli-X/Y/Z rotation mean Rx/Ry/Rz rotations. do you mean rotation Rx(proportional to feature value)?

Reply:
I renamed them RX, RY, RZ, which is the PL standard. Hope that is clearer!